airflow/gcp/sensors/bigtable.py

```diff
@@ -31,7 +31,7 @@
 from airflow.utils.decorators import apply_defaults
 
 
-class BigtableTableWaitForReplicationSensor(BaseSensorOperator, BigtableValidationMixin):
+class BigtableTableReplicationCompletedSensor(BaseSensorOperator, BigtableValidationMixin):
```

Comment:
I think we should change reference in L44:
:ref:`howto/operator:BigtableTableWaitForReplicationSensor`

same goes for update of how-to guide in docs.

Reply:
Thanks, I will fix it.